Libraries/StyleSheet/StyleSheet.js

```diff
@@ -17,6 +17,9 @@ var StyleSheetValidation = require('StyleSheetValidation');
 
 var flatten = require('flattenStyle');
 
+export type Styles = {[key: string]: Object};
```

Comment:
@ide Not exactly sure how Flow works here. Do arrays also count as objects? If not then they are same :)